workshops/konami_code/README.md

```diff
@@ -0,0 +1,478 @@
+
+It will look something like this after someone types the konami code:
+
+[![Image](https://cloud-5apj3kt1e.vercel.app/image.png)](https://konami-code.faisalsayed1.repl.co/)
+
+Here's the [live demo](https://konami-code.faisalsayed1.repl.co/)  
+Here's the [source code](https://repl.it/@FaisalSayed1/Konami-Code)
+
+Live demo: 
+![Image](https://cloud-95l2730mf.vercel.app/konamigif.gif)
+
+## Part 1: Prerequisites
+
+You should have a beginner understanding of:
+
+- HTML
+- CSS
+- JavaScript
+
+Also for this workshop, we will need a library called 'confetti.js'. 
+
+Download it by going [here](https://cloud-72a7nfmm7.vercel.app/index.min.js). Right click and select 'save as'.
+
+## Part 2: Setup
+
+### Setting up your code environment on Repl.it
+
+[Repl.it](https://repl.it) is an online code editor. You don't have to use Repl.it but I suggest you do as it sets everything up for you and you don't require any installations.
+
+To get started, go to [https://repl.it/languages/html](https://repl.it/languages/html). Your coding environment will spin up in just a few seconds!
+
+You should see something like the following:
+
+![Image](https://cloud-p7qnbqzo6.vercel.app/image.png)
+
+Import the confetti.js (`index.min.js`) file which you downloaded on the left side of the screen.
+
+![Image](https://cloud-2d72c0n78.vercel.app/indexminjs.gif)
+
+## Part 3: Building the project
+
+### 1) HTML
+
+Let's start writing the HTML code required in the `index.html` file. First, we'll import the required JavaScript files.
+
+Inside the `<head>` tag, we'll import the [`anime.js`](https://animejs.com/) library. Copy the code given below.
+
+```html
+<script src="https://cutt.ly/anime_js"></script>
+```
+
+
+Also link the `index.min.js` file to our HTML inside the `<head>` tag.
+
+```html
+<script src="index.min.js"></script>
+```
+
+The code so far:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style.css" rel="stylesheet"
+    type="text/css" />
+    <script src="https://cutt.ly/anime_js"></script>
+    <script src="index.min.js"></script>
+  </head>
+  <body>
+    <script src="script.js"></script>
+  </body>
+</html>
+```
+
+Next, we'll add some basic HTML code inside of our `<body>` tag.
+
+```html
+<canvas id="my-canvas"></canvas>
+<h1 class='text'>Password Please</h1>
+<div class="secret-div">
+  <h1>Hello comrade!</h1>
+</div>
+```
+
+**Note:** Make sure the `<script>` tag, which is in your body, is just above the closing `</body>` tag and whatever code you write in the `<body>` should be above that `<script>` tag or else it will throw you errors.
+
+**Explanation:** We are adding a canvas element which will be later used for the confetti. Then there is also a `secret-div`, it will be initially hidden but when someone types the konami code, we'll make it visible.
+
```

Comment:
```suggestion
**Note:** Make sure the `<script>` tag, which is in your body, is just above the closing `</body>` tag and whatever code you write in the `<body>` should be above that `<script>` tag or else it will throw you errors.

**Explanation:** We are adding a canvas element which will be later used for the confetti. Then there is also a `secret-div`, it will be initially hidden but when someone types the Konami code, we'll make it visible.

```

Reply:
same here too. why?